src/reporter.js

```diff
@@ -191,15 +210,18 @@ function CypressXML(runner, options) {
         testCases.push(createTestRecord(t, specRelativePath));
         if (t.state == 'failed') suiteStats.failures++;
       })
-
-      var logFile = path.join(config.logsFolder, specRelativePath).replace('.js', '.txt');
       var logContent = '';
-      if (fs.existsSync(logFile)) {
-        logContent = fs.readFileSync(logFile, 'utf8');
+      if (config.logsFolder !== null) {
+        var logFile = path.join(config.logsFolder, specRelativePath).replace('.js', config.logFileExt);
+        if (fs.existsSync(logFile)) {
+          logContent = fs.readFileSync(logFile, 'utf8');
```

Comment:
As I mentioned elsewhere, need to do "binary" reading - expecting `utf8` encoded could lead to errors